src/types/types.js

```diff
@@ -524,6 +524,12 @@ export type EdgeCurrencyEngine = {
 
 // currency plugin -----------------------------------------------------
 
+export type EdgeParseUriOptions = {
```

Comment:
Make this a cleaner and export it for use by the plug-ins. It will clean up the code that tries to validate the options